src/rules/sort-keys.js

```diff
@@ -107,7 +107,9 @@ const rule = {
 		],
 
 		docs: {
+			recommended: true,
```

Comment:
```suggestion
			recommended: false,
```

This rule is not enabled in the `recommended` config:

https://github.com/eslint/json/blob/c99f2ad8ec23087e53cb6da9dcfab11ada0ddc53/src/index.js#L41-L51